requirements.txt

```diff
@@ -1,7 +1,7 @@
 certifi
 configparser==3.5.0
 coverage==4.2
-cryptography==1.5.2
+cryptography==1.7.2
```

Comment:
Are we going to have the same issue with 1.7.2 as we did with 1.5.2, on some python / os versions?

Should we not just make this ==1.x or >= 1.7?